package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.24.5",
+    "version": "25.0.0",
     "description": "React components for Clarity UI",
     "license": "Apache-2.0",
     "private": false,
     "outDir": "dist",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "dependencies": {
-        "@clr/icons": "3.0.0",
-        "@clr/ui": "3.0.0",
+        "@clr/icons": "4.0.8",
```

Comment:
This is not related to this PR. Same changes are needed in vsphere UI repo.
"@clr/icons": "4.0.8",
"@clr/ui": "4.0.8",

Reply:
@payald We are going to take care of this in different PR once this PR gets merged